test/unit/facts.test.js

```diff
@@ -650,6 +651,25 @@ tap.test('display_host', { timeout: 20000 }, (t) => {
     })
   })
 
+  t.test('should be process.env.DYNO when use_heroku_dyno_names is true', (t) => {
```

Comment:
good work finding where these tests should go. I was a little caught off guard but realized this is where the tests live for this function 🤷🏻